app/services/partner_routing_service.rb

```diff
@@ -18,7 +18,7 @@ def determine_partner
     from_state_routing = vita_partner_from_state if @zip_code.present?
     return from_state_routing if from_state_routing.present?
 
-    route_to_national_overflow_partner
+    nil
```

Comment:
`route_to_national_overflow_partner` sets a `routing_method` - i wonder if we should add one like `not_routed` or `none` to leave a paper trail so we know which clients were not assigned a partner due to capacity problems.

i should probably mention that this change will be coupled with removing the "continue anyway" link on the at capacity page, so no one will be getting past this and therefore should not get lost in the hub (i think? at this point their status should be something that makes them invisible in the hub?)

however, even assuming no one gets past the at capacity page, is it a bad idea to assign these clients to no one? my thinking right now is that it's okay because they won't have entered contact info and therefore won't be able to come back as a "returning client"

Reply:
you could add a routing method called "over_capacity" or something like that

Reply:
the default return of a method is nil, so you don't even really have to explicitly return nil like this. I'd probably go with commenting out national routing line and adding a new line that sets the routing method to "none" or "over_capacity"

Reply:
ah good point about it already returning nil. funny thing is that if i add the line for `@routing_method = :at_capacity` it no longer returns nil 😆